app/assets/stylesheets/markdown-editor.scss

```diff
@@ -0,0 +1,91 @@
+.md-editor {
+  .md-footer,
+  .md-header {
+    background: $sidebars-background;
+    border: 0;
+    color: $text;
+    display: block;
+    height: 42px;
+    margin: 0;
+    padding: 6px 6px 0;
+  }
+
+  .md-header,
+  .nav-tabs {
+    border-bottom: 1px solid $border-grey;
+    margin: 0;
+  }
+
+  .md-cancel {
+    color: $text-grey;
+
+    &:hover { color: $text; }
+
+    .fa-times { font-size: 13px; }
+  }
+
+  .md-preview {
+    background: $white;
+    color: $text-color;
+    // !important is needed to override the CSS rules dynamically added to the element
+    // scss-lint:disable ImportantRule
+    height: auto !important;
+    // scss-lint:enable ImportantRule
+    min-height: 90px;
+    overflow: auto;
+    padding: 6px 12px;
+    position: relative;
+    // !important is needed to override the CSS rules dynamically added to the element
+    // scss-lint:disable ImportantRule
+    width: 100% !important;
+    // scss-lint:enable ImportantRule
+    z-index: 10;
+  }
+
+  .md-controls {
+    float: right;
+    padding: 3px;
+
+    .md-control {
+      color: $text-grey;
+      padding: 3px 3px 3px 10px;
+      right: 5px;
+    }
+  }
+
+  .btn-group:not(.nav-tabs) {
+    margin-bottom: 6px;
+  }
+}
+
+.write-preview-tabs {
+  &,
+  & .full-height {
+    height: 36px;
+  }
+
+  > li > a { padding: 7px 15px; }
+
+  a:focus { outline: none; }
+
+  li {
+    &:not(.active) a:focus,
+    &:not(.active) a:hover {
+      background-color: transparent;
+      border-left: 1px solid transparent;
+      border-right: 1px solid transparent;
+      border-top: 1px solid transparent;
+    }
+
+    &.active:focus { color: $black; }
+  }
+
+  .tab-help-text { margin-left: 5px; }
+
+  .diaspora-custom-compose:before {
+    bottom: -2px;
+    position: relative;
+  }
+}
+
+#publisher_textarea_wrapper:not(.active) .md-header { display: none; }
```

Comment:
Selector `publisher_textarea_wrapper` should be written in lowercase with hyphens